model/clusters_mgmt/v1/cluster_autoscaler_type.model

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Cluster-wide autoscaling configuration.
+struct ClusterAutoscaler {
```

Comment:
Can you break these structs into their own files please, its rare we bundle them like this.

Reply:
sure. done